openfisca-web-api/input-output-data.md

```diff
@@ -1,104 +1,294 @@
+ - A formula or a default value, that are used for computations when the input variable was not provided.
 
-A JSON scenario is an object structured this way:
-* `axes` (a list of objects, default: null): the axes of the scenario, see [axes](#axes)
-* `input_variables` (an object, mutually exclusive with `test_case`): the input variables, encoded with the [value by period](#values-by-period) structure. For instance:
-  * `{"salary": {"2015-04": 2000}, age: {"2015-04": 30}}`
-* `period` (see [periods and instants](../periodsinstants.md)): the period on which the variables of the decomposition will be computed
-* `test_case` (an object, mutually exclusive with `input_variables`): the test case of the scenario, see [test cases](#test-cases)
 
-> Either `test_case` or `input_variables` must be provided, not both.
+To provide an input variable, insert the value in the json, for the corresponding time period and entity.
 
-> `axes` can't be used with `input_variables`, only `test_case`.
+> For example, if Ricarda has a salary (defined monthly for a Person) of 3500/month until september 2016, and 4000/month after that and if household_2 were tenant and became homeowners in march 2016 (housing_occupancy_status is defined monthly for a household) of the 57 sqm apartment they live in, you would write:
 
-## Test cases
-
-A test case describes persons, entities and their associations.
+```json
+{
+  "persons": { 
+    "Ricarda": {
+      "salary": {
+        "2016-01": 3500,
+        "2016-02": 3500,
+        "2016-03": 3500,
+        "2016-04": 3500,
+        "2016-05": 3500,
+        "2016-06": 3500,
+        "2016-07": 3500,
+        "2016-08": 3500,
+        "2016-09": 4000,
+        "2016-10": 4000,
+        "2016-11": 4000,
+        "2016-12": 4000
+      }
+    },
+    "Bob": {},
+    "Bill": {},
+    "Janet": {}
+  },
+  "households": {
+    "household_1": {
+      "parents": [
+        "Ricarda", "Bob"
+      ]
+    },
+    "household_2": {
+      "parents": [
+        "Bill"
+      ],
+      "children":[
+        "Janet"
+      ],
+      "housing_occupancy_status": {
+        "2016-01": "Tenant",
+        "2016-02": "Tenant",
+        "2016-03": "Owner",
+        "2016-04": "Owner",
+        "2016-05": "Owner",
+        "2016-06": "Owner",
+        "2016-07": "Owner",
+        "2016-08": "Owner",
+        "2016-09": "Owner",
+        "2016-10": "Owner",
+        "2016-11": "Owner",
+        "2016-12": "Owner"
+      },
+      "accommodation_size": {
+        "2016-01": 57,
+        "2016-02": 57,
+        "2016-03": 57,
+        "2016-04": 57,
+        "2016-05": 57,
+        "2016-06": 57,
+        "2016-07": 57,
+        "2016-08": 57,
+        "2016-09": 57,
+        "2016-10": 57,
+        "2016-11": 57,
+        "2016-12": 57
+      }
+    }
+  }
+}
+```
+> note that because of the default value system in OpenFisca, the variables that have not been defined explicitly are either calculated or take on their default value.  
```

Comment:
Not sure this should be greyed and added as a side node. This is a quite important thing to understand.